app.js

```diff
@@ -0,0 +1,112 @@
+var restler = require('restler');
+var uuid = require('node-uuid');
+var sslRootCAs = require('ssl-root-cas/latest');
+sslRootCAs.inject();
+var fs = require('fs');
+var request = require('request');
+var async = require('async');
+var express = require('express');
+var app = express();
+var multer = require('multer');
+var upload = multer({dest:'./uploads/'});
+var exec = require('child_process').exec;
+var cmd = '/usr/games/fortune | /usr/games/cowsay';
+var Constants = require(__dirname + '/Constants.js')
+
+var stlFilePath = "./3dFiles/original.stl";
+var myRes = null;
```

Comment:
Since all variables in the global space are shared between requests, this will break if you have more than 1 request running at the same time.  Imagine this:

1. We get a POST /image request, which we will call A.  We start creating the HTTP response for A, which we store in the global myRes object with `myRes = res`
2. While we're busy processing A's response (doing POST Constants.UPDATE_SCENE_ENDPOINT, etc.), we get another request B for POST /image.
3. When we call the route for B, we'll do `myRes = res`, overwriting A's response with B's response.
4. When A is done and ready to respond (`myRes.status(err.status || 500).end();`), we instead close off B's response, giving B A's response data.  A's request is never responded to.